### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by weilingooi**
ready
rubocop

### CODE REVIEW

**Comment by weilingooi** [comment]
ready

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1 @@
+defaults
```
Comment (saobv-2223) [inline .browserslistrc]: Add this file to .gitignore and run `git rm --cached .browserlistrc` to untrack this file

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,56 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (saobv-2223) [inline Gemfile]: please, remove all comment line and sort gem file A to Z

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", "~> 1.4"
```
Comment (saobv-2223) [inline Gemfile]: Use mysql2 instead of sqlite3

File: app/javascript/packs/application.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: In js and css, we use ' 
In ruby file, we use "

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: Use ", please

File: config/database.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
```
Comment (saobv-2223) [inline config/database.yml]: You shoud create database.yml.example and push to git, that has same code as database.yml.
Add database.yml to gitignore and run git rm --cached config/database.yml.

File: test/application_system_test_case.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
+end
```
Comment (saobv-2223) [inline test/application_system_test_case.rb:5]: We don't use this test folder in our project, so add this folder to .gitignore and run `git rm --cached -r test`

**Review by saobv-2223 — COMMENTED** [review]
This pull is having 2 commits, merge them to ONE commit. Notes: A pull request only has ONE commit

File: .browserslistrc (hunk 1 above)
Comment (vanvtt-0952) [reply .browserslistrc]: you don't fix it

File: app/javascript/packs/application.js (hunk 4 above)
Comment (vanvtt-0952) [reply app/javascript/packs/application.js]: you need this comment
you use `"` to replace `'`

File: app/mailers/application_mailer.rb [hunk 8]
```diff
@@ -0,0 +1,6 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
+  layout "mailer"
+  default from: "from@example.com"
+  layout "mailer"
```
Comment (vanvtt-0952) [inline app/mailers/application_mailer.rb]: duplicate code

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (vanvtt-0952) [inline app/views/layouts/application.html.erb]: using `"` replace for `'`

File: config/database.yml (hunk 6 above)
Comment (vanvtt-0952) [reply config/database.yml]: You shoud create database.yml.example and push to git, that has same code as database.yml.
Add database.yml to gitignore and run git rm --cached config/database.yml.

File: Gemfile [hunk 10]
```diff
@@ -0,0 +1,96 @@
+<<<<<<< HEAD
```
Comment (vanvtt-0952) [inline Gemfile]: you fix the faulty conflict code

File: Gemfile.lock [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+    childprocess (3.0.0)
+    concurrent-ruby (1.1.6)
+    crass (1.0.6)
+    erubi (1.9.0)
+    ffi (1.13.1)
+    globalid (0.4.2)
+      activesupport (>= 4.2.0)
+    i18n (1.8.5)
+      concurrent-ruby (~> 1.0)
+<<<<<<< HEAD
```
Comment (vanvtt-0952) [inline Gemfile.lock]: plz, delete the content of this file, then you run bundle

File: app/jobs/application_job.rb [hunk 12]
```diff
@@ -0,0 +1,11 @@
+class ApplicationJob < ActiveJob::Base
+  # Automatically retry jobs that encountered a deadlock
+  # retry_on ActiveRecord::Deadlocked
+
+<<<<<<< HEAD
+  # Most jobs are safe to ignore if the underlying records are no longer availa
+=======
+  # Most jobs are safe to ignore if the underlying records are no longer available
+>>>>>>> d3fb23d... Init Project
```
Comment (vanvtt-0952) [inline app/jobs/application_job.rb]: plz, fix conflict

File: app/mailers/application_mailer.rb [hunk 13]
```diff
@@ -0,0 +1,9 @@
+class ApplicationMailer < ActionMailer::Base
+<<<<<<< HEAD
+  default from: "from@example.com"
+  layout "mailer"
+=======
+  default from: 'from@example.com'
+  layout 'mailer'
+>>>>>>> d3fb23d... Init Project
```
Comment (vanvtt-0952) [inline app/mailers/application_mailer.rb]: you fix the faulty conflict code

Comment (vanvtt-0952) [reply app/mailers/application_mailer.rb]: you should remove line 2, 5, 6, 7, 8

**Comment by vanvtt-0952** [comment]
Merged
This is ok. You could merged this pull request

**Pull request merged**